packages/components/src/edit-in-place-control/style.scss

```diff
@@ -0,0 +1,11 @@
+.components-edit-in-place-control__label {
+	padding: $grid-unit-30;
+	width: 100%;
+}
+.components-edit-in-place-control__input {
+	margin-bottom: 1px;
+	text-indent: $grid-unit;
+}
+.block-editor-block-toolbar-animated-width-container {
+	width: auto !important;
```

Comment:
Is the `!important` really necessary?

Reply:
Umm this seems to be some rebase leftover?